transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVParser.java

```diff
@@ -189,6 +212,11 @@ public void transform(StructuredRecord in, Emitter<StructuredRecord> emitter) th
     // Field has to string to be parsed correctly. For others throw an exception.
     String body = in.get(config.field);
 
+    // Handles case where nullable input field was null
+    if (body == null) {
```

Comment:
lets add a short blurb in the field description and in the .md doc file that records with a null field get filtered out.